setup.cfg

```diff
@@ -29,6 +29,7 @@ install_requires =
     qtterm
     requests[socks]
     tomlkit
+    darkdetect-angr[macos-listener]
```

Comment:
* It's unfortunate that we have to maintain a forked release of darkdetect now, but I appreicate the work on it
* Did Windows/Linux get tested?

Reply:
The majority of the fork has a PR into the main repo, so once that is merged we might not need to hopefully. TBD. I've tested the PR'd version on all three OS's, though less throughly on Windows as I lack said OS. The version we are using is mostly the same so it ought to work. Feel free to test them yourself but the relevant logic that distinguished between OSes wasn't really the code that changed between the PR'd version and the hardfork version.

Reply:
> I've tested the PR'd version on all three OS's

But not the forked version with extra changes?

> Feel free to test them yourself

I don't plan to test this PR. Will you please test it

Reply:
The extra changes are copyright, README, and `__version__`, and `pyproject.toml` to change the package name. Not really the code.

Reply:
If you are curious, here is the diff stats:
```
 LICENSE                     |  4 ++--
 README.md                   |  8 ++++----
 darkdetect/__init__.py      |  4 ++--
 darkdetect/__main__.py      |  2 +-
 darkdetect/_dummy.py        |  2 +-
 darkdetect/_linux_detect.py |  2 +-
 docs/api.md                 |  2 +-
 docs/examples.md            |  2 +-
 pyproject.toml              | 10 +++++-----
 ```
 Note the 2 line changes are copyright comments. I don't have access to a Windows device at the moment so I can't really test these changes easily, but they aren't really code changes just the changes necessary to hard-fork a project (changing the name on pypi, copyright, documentation, and version).

Reply:
Github actions supports windows, why not write some test cases?

Reply:
I do not know how to automate GUI test cases on windows changing system theme and detecting of the colors properly all changed over. Feel free. The actions to test this would be launch app, open preferences, select 'Track System Theme', then toggle the Windows System Theme and visually ensure it tracks the theme.

Reply:
You don't need to "visually ensure" it, you just need to make sure after the theme is changed the correct behavior occurs in the library. I'm sure applescript and powershell each have ways to adjust the system theme.

Reply:
That's on the angr management side, but not necessary for the library itself which is is the subject of this thread.

Reply:
I'll happily take a PR for them here: https://github.com/zwimer/darkdetect but I don't have plans to write tests myself for them at the moment. If you do PR them, I'll also merge them into the branch that is PR-ing into the original repo https://github.com/albertosottile/darkdetect so as to avoid actively diverging the hard fork from the original more than necessary; my plan is to get rid of the fork once the PR https://github.com/albertosottile/darkdetect/pull/32 in the original is merged.

Right now that PR has been tested on all three OSes and my fork's master branch doesn't meaningfully diverge from said code. I know the maintainer desired to do more thorough testing before merging the PR though, so I'm sure the test cases would be appreciated by all parties.